sonar-python-plugin/src/main/java/org/sonar/plugins/python/IPythonNotebookParser.java

```diff
@@ -78,4 +82,23 @@ private static void processCodeCell(JsonParser parser) throws IOException {
       }
     }
   }
+   private static Map<Integer, Integer> countEscapeCharacters(String sourceLine, Map<Integer, Integer> colMap, int colOffSet) {
+    int count = 0;
+    var arr = sourceLine.toCharArray();
+    for (int i = 1; i < sourceLine.length() - 1; ++i) {
+      char c = arr[i];
+      switch (c) {
+        case '"','\'', '/': 
```

Comment:
Is `/` an escaped character in JSON string? I'm wondering why this one appears here.
In any case, I guess this category concerns characters that are escaped when converting the JSON string to a Java string, right?

Reply:
Yes exactly. This list is not exhaustive at all and ha to be confirmed.